pkgs/development/compilers/openjdk/corretto.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, bash
+, openjdk8
+
+, ...
```

Comment:
Nit: Without args I think it makes sense to remove the rest capture as well

Reply:
Yep, also fair - done! :+1: